src/Cronofy.php

```diff
@@ -845,6 +845,18 @@ public function realTimeScheduling($params)
         return $this->httpPost("/" . self::API_VERSION . "/real_time_scheduling", $postFields);
     }
 
+    public function realTimeSchedulingDisable($params)
+    {
+
+        $postFields = [
+            'client_secret' => $this->clientSecret,
```

Comment:
I don't think the `client_secret` and `id` go in the request body: [docs](https://docs.cronofy.com/developers/api/scheduling/real-time-scheduling/disable/#example-request)
You're correctly using the ID in the URL on :857, but need to use the `apiKeyHttpPost` function for the helper which POSTs with the API key header.